fctesting/utils.go

```diff
@@ -43,3 +46,24 @@ func RequiresRoot(t testing.TB) {
 			rootDisableEnvName)
 	}
 }
+
+func newLogger() *log.Logger {
+	str := os.Getenv(logLevelEnvName)
+	if str != "" {
+		logLevel, err := log.ParseLevel(str)
+		if err != nil {
+			panic(err)
```

Comment:
Would suggest passing in `testing.TB` and calling `Fatalf` instead. This allows users to know that this is only to be used in testing as well.

Reply:
Thanks. Will do.